services/user.py

```diff
@@ -0,0 +1,48 @@
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = "",
+    last_name: str = ""
+) -> User:
+    user = User.objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name
+    )
+    return user
+
+
+def get_user(user_id: int) -> User:
+    user = User.objects.get(id=user_id)
+    return user
```

Comment:
Use get_user_model() instead of User. Check out the [checklist](https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency).